test/extended/authentication.sh

```diff
@@ -65,27 +65,35 @@ wait_for_command 'oc get imagestream openldap --template="{{with \$tags := .stat
 # kick off a build and wait for it to finish
 oc start-build openldap --follow
 
-# wait for the deployment to be up and running
-wait_for_command 'oc get pods -l deploymentconfig=openldap-server --template="{{with \$items := .items}}{{with \$item := index \$items 0}}{{\$item.status.phase}}{{end}}{{end}}" | grep Running' $((60*TIME_SEC))
+# wait for LDAP server to be ready
+wait_for_command 'oc get pods -l deploymentconfig=openldap-server --template="{{with \$items := .items}}{{with \$item := index \$items 0}}{{with \$map := index \$item.status.conditions 0}}{{index \$map \"type\"}}{{end}}{{end}}{{end}}" | grep Ready' $((60*TIME_SEC)) 
```

Comment:
umm... wow.  That's an impressive template.  Add a TODO try jsonpath and see if its better.